demo/client/backend/mock/README.md

```diff
@@ -0,0 +1,35 @@
+# Start with go chaincode
+
+    $ go run .
+
+# Start with fpc chaincode
+
+Note that you need to start the docker container from the fpc root
```

Comment:
You don't really need docker for fpc either, so for simplicity and consistency perspective it probably is better to drop the docker part here as well (essentially, remove first paragraph).

That relates also to the comments for the makefile. Maybe better/easier to have makefile targets which directly build and run fpc mode without docker? In fact, ideally we might have two targets, 'run-go' and 'run-fpc'?

Reply:
> You don't really need docker for fpc either, so for simplicity and consistency perspective it probably is better to drop the docker part here as well (essentially, remove first paragraph).

Ok, let's drop the docker part here.

> That relates also to the comments for the makefile. Maybe better/easier to have makefile targets which directly build and run fpc mode without docker? In fact, ideally we might have two targets, 'run-go' and 'run-fpc'?

Will add that